src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-put.test.ts

```diff
@@ -0,0 +1,186 @@
+import { Installation } from "models/installation";
+import express, { Express, NextFunction, Request, Response } from "express";
+import { RootRouter } from "routes/router";
+import supertest from "supertest";
+import { getLogger } from "config/logger";
+import { encodeSymmetric } from "atlassian-jwt";
+import { GitHubServerApp } from "models/github-server-app";
+
+describe("PUT /jira/connect/enterprise/app/:uuid", () => {
+	let app: Express;
+	let installation: Installation;
+	let jwt: string;
+	const uuid = "4c74e9ce-faf9-489e-821f-e7a8006e473a";
+
+	beforeEach(async () => {
+		installation = await Installation.install({
+			host: jiraHost,
+			sharedSecret: "shared-secret",
+			clientKey: "client-key"
+		});
+
+		app = express();
+		app.use((req: Request, res: Response, next: NextFunction) => {
+			res.locals = { installation };
+			req.log = getLogger("test");
+			req.session = { jiraHost };
+			req.params = { uuid };
+			next();
+		});
+		app.use(RootRouter);
+
+		jwt = encodeSymmetric({
+			qsh: "context-qsh",
+			iss: jiraHost
+		}, installation.sharedSecret);
+	});
+
+	it("should return 200 with success true when correct uuid and installation id are passed", async () => {
```

Comment:
For `uuid and installation id are passed` <-- I don't think `installation id` is passed (from client browser to server)?

Reply:
It's not. The installation is stored in res.locals.